tests/test_black.py

```diff
@@ -10,12 +11,16 @@
 import black
 
 ll = 88
-ff = partial(black.format_file, line_length=ll, fast=True)
 fs = partial(black.format_str, line_length=ll)
 THIS_FILE = Path(__file__)
 THIS_DIR = THIS_FILE.parent
 
 
+def ff(file_):
+    with tokenize.open(file_) as buf:
+        black.format_file(buf, line_length=ll, fast=True)
```

Comment:
This is fine. Also, you can just use `file`. The builtin name `file` is deprecated, we only keep it to not break old code.